exporters/trace_stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverExporter.java

```diff
@@ -72,13 +73,13 @@ public static void createAndRegisterWithCredentials(Credentials credentials, Str
    * Stackdriver exporter can be registered at any point.
```

Comment:
Suggestion: I think it would be better to include how the method differs from the others in the summary fragment, since that is the only part that is shown in some places.  For example, this one could say "Creates and registers the Stackdriver Trace exporter to the OpenCensus library for an explicit project ID."

Reply:
Done.